### PR TITLE
fix: module not found in @scaleleap/amazon-marketplaces

### LINKED ISSUE #1023: code: 'MODULE_NOT_FOUND'   requestPath: '@scaleleap/amazon-marketplaces'
When I encountered an error while using this SDK in a new project, it may be related to the **scaleleap/amazon-marketplaces** version. The current package dependencies are as follows. I would appreciate it if you could make the necessary adjustments. Thank you very much.

├─┬ @scaleleap/amazon-advertising-api-sdk@3.8.3
│ └── @scaleleap/amazon-marketplaces@17.3.1

```
`  code: 'MODULE_NOT_FOUND',
  path: '/XXXXX/node_modules/@scaleleap/amazon-advertising-api-sdk/node_modules/@scaleleap/amazon-marketplaces/package.json',
  requestPath: '@scaleleap/amazon-marketplaces'`
```

Mabye related this issue https://github.com/ScaleLeap/amazon-marketplaces/issues/634

### PR DESCRIPTION
**Pull request by nguyentoanit**
There was a bug in the `@scaleleap/amazon-marketplaces` package.
Need to update to the latest version to fix this

Closes #1023

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@scaleleap/amazon-marketplaces": "^17.1.0",
+    "@scaleleap/amazon-marketplaces": "18.0.1",
```
Comment (moltar) [inline package.json:34]: Probably should be a caret ^18

**Review by moltar — CHANGES_REQUESTED** [review]

**Comment by jiangdi0924** [comment]
Could you please let me know when this can be merged into the main branch? Thank you.

**Comment by moltar** [comment]
@nguyentoanit Please wrap this up asap.

**Comment by nguyentoanit** [comment]
@moltar @jiangdi0924 : We got an issue when release 🐞: https://github.com/ScaleLeap/amazon-advertising-api-sdk/actions/runs/10805243502/job/29972026362

I'm trying to fix it soon.

**Pull request merged**